### PR TITLE
Add some missing periods to the readme

### PR DESCRIPTION
**Pull request by mheon**
Really I just want to fire the CI. But add some missing periods to make this section of the readme consistent.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ At a high level, the scope of libpod and Podman is the following:
 * Support multiple image formats including the OCI and Docker image formats.
 * Support for multiple means to download images including trust & image verification.
 * Container image management (managing image layers, overlay filesystems, etc).
-* Full management of container lifecycle
-* Support for pods to manage groups of containers together
+* Full management of container lifecycle.
+* Support for pods to manage groups of containers together.
```
Comment (TomSweeneyRedHat) [inline README.md]: Sniff, I'm so happy with this change!  ;^)

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests.

**Pull request merged**